packages/@sanity/language-filter/src/datastore.ts

```diff
@@ -63,7 +63,8 @@ const filterField = config.filterField || defaultFilterField
 
 export const filterFn$ = selectedLanguages$.pipe(
   map((langs) => {
-    return (enclosingType: SchemaType, field: ObjectField) =>
-      filterField(enclosingType, field, langs)
+    return (enclosingType: SchemaType, field: ObjectField) => {
```

Comment:
this diff doesn't seem relevant? can it be (re)moved?

Reply:
Same as above re. `filterField`